### DISCUSSION TITLE
Proposal for Adding "Uptane Contributors" Section to the Website and Workflow Integration

### DISCUSSION THREAD

**Discussion by Abhijay007** [uptane/uptane.github.io#144] [Ideas]
Hello Uptane community,

We are considering the addition of a new section on our website dedicated to "Uptane Contributors." Here's a preview of how it would appear:

Uptane Contributors

![image](https://github.com/uptane/uptane.github.io/assets/64387054/bdf3fd9a-42e6-479f-8167-6a0d4936d3d8)

To streamline the process, we are seeking a solution that automates triggers or workflows whenever a new contributor is added. Importantly, we aim to feature not only contributors to the "uptane website contributors" but also contributors from across the entire Uptane community projects on github.

If anyone is aware of an open-source solution that fits our requirements, please share your insights. As a reference, the Moja Global repository has implemented a similar feature on their community website. You can explore it here: [Moja Global Community](https://community.moja.global/) (scroll down to the "Contributors" section).  you can find the corresponding repository on GitHub: [Moja Global Community Website Repo](https://github.com/moja-global/community-website).

We appreciate any feedback or suggestions you may have on implementing a comparable solution for our community repository.

**Comment by ashmitjsg** [comment]
We can use a combination of Github Workflow and a script file to achieve this. [Moja Global Community Website](https://github.com/moja-global/community-website) uses a [javascript script](https://github.com/moja-global/community-website/blob/main/website/scripts/fetchContributors.js#L22) to fetch all the contributors for its website repository and run it once every day using [GitHub Workflow](https://github.com/moja-global/community-website/blob/main/.github/workflows/update-contributors.yml). 

Since in our case we want to display contributors from all the repositories of our org, we have to use a different script. Moreover, we can also use bash script instead of javascript to achieve this. I can do this in both ways. Once, we have the list of contributors in a `contributors.md` file we can display it on our website easily and it will automatically get updated every day by GitHub workflow.

**Reply by Abhijay007** [reply]
so do you want to give it a shot?

**Reply by ashmitjsg** [reply]
@Abhijay007, yeah sure, I will. I am currently traveling so will do this on 3rd December. I have the code and workflow almost ready for this.

**Reply by Abhijay007** [reply]
sure, take your time :)

**Reply by ashmitjsg** [reply]
Hi @Abhijay007, should I directly create a PR for this or first open an issue?

**Comment by Abhijay007** [comment]
> Hi @Abhijay007, should I directly create a PR for this or first open an issue?

@devilkiller-ag I created a issue for the same

**Comment by hexsecs** [comment]
This was completed with #163
